.travis/install.sh

```diff
@@ -91,7 +87,7 @@ VARSYAML
 if [[ "$TEST" == "pulp" || "$TEST" == "performance" || "$TEST" == "s3" ]]; then
   sed -i -e '/^services:/a \
   - name: pulp-fixtures\
-    image: docker.io/pulp/pulp-fixtures:latest\
+    image: docker.io/pulp/pulp-fixtures:latest
```

Comment:
This is causing the test failures. I just merged a PR to fix it.

https://github.com/pulp/plugin_template/pull/236/